src/lang.js

```diff
@@ -11,8 +11,25 @@ const messages = {
   da
 }
 
+const dateTimeFormats = {
+  'da': {
```

Comment:
No english formats needed? Maybe the are default?

Reply:
I made english, and yes it was required, but weekday is still written in danish because of OS locale I guess.

Reply:
Odd with the weekday. It should adhere to locale set in app and not OS locale. Otherwise good stuff.